sources/packages/backend/apps/api/src/route-controllers/supporting-user/supporting-user.supporting-users.controller.ts

```diff
@@ -141,7 +141,7 @@ export class SupportingUserSupportingUsersController extends BaseController {
   ): Promise<void> {
     // Regardless of the API call is successful or not, create/update
     // the user being used to execute the request.
-    const userQuery = this.userService.syncUser(
+    const userQuery = await this.userService.syncUser(
```

Comment:
Line 144 and 154 are expected to be promise which is resolved in line 161 (Promise.all)

Reply:
Yes, you're right. Thanks. I've trying something and forgot to remove that.